scipy/linalg/flapack.pyf.src

```diff
@@ -735,11 +763,296 @@ interface
           :: lwork=2*minmn+MAX(maxmn,nrhs)
           ! check(lwork>=2*minmn+MAX(maxmn,nrhs))
```

Comment:
This (existing) check on the lwork size is probably more exact than is needed and is more exacting than that added below for ?gelsy.  Do we want this kind of check or should we just anything through and let lapack complain?